stack-overflow/src/view/QuestionView.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import '../styles/questionStyle.css';
+
+const QuestionView = ({ questions, onCreate, onChange, onChangeQuestion, onFilterByTitle, onFilterByTags, searchFieldTitle, searchFieldTag, showQuestions, logOut, title, text, tags, allTags }) => (
```

Comment:
This view is too large. Think about splitting it up into several smaller dumb components (that you could then compose into a single dumb component).